media-gfx/openexr_viewers/openexr_viewers-2.2.0-r1.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit cmake-utils flag-o-matic
+
+DESCRIPTION="OpenEXR Viewers"
+HOMEPAGE="http://openexr.com/"
+# changing sources. Using a revision on the binary in order
+# to keep the old one for previous ebuilds.
+SRC_URI="https://github.com/openexr/openexr/archive/v${PV}.tar.gz -> openexr-${PV}-r1.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+IUSE="cg namespace-versioning"
+
+RDEPEND="~media-libs/ilmbase-${PV}:=[namespace-versioning=]
+	~media-libs/openexr-${PV}:=[namespace-versioning=]
```

Comment:
If unset, the name will be library.so.22 (api number)

If set, the name will be library-2_2.so.22 (the version and api number)

If API doesn't change then the programs depending on this library will not trigger a recompile to link to the new ones. Since the API is the same, that means bug fixes under the hood.